ui/pages/users/add.js

```diff
@@ -123,6 +123,16 @@ export default function UsersAdd() {
           errors[error.fieldName.toLowerCase()] =
             error.errors[0] || 'invalid value'
         }
+
+        // TODO: need to work with backend for better error message
+        if (
+          e.code === 409 &&
+          errors['identity_id'] &&
+          errors['identity_id'].includes('already exist')
```

Comment:
Error 409 implies a conflict so I think we don't have to check for this string (which could be flaky)